tests/__sf_section_user.test.fish

```diff
@@ -4,6 +4,10 @@ function setup
 	spacefish_test_setup
 end
 
+function teardown
+	set USER $LOGNAME
```

Comment:
This will reset the user to be the username that initially logged in.
It seems this wasn't being reset after setting user to "ROOT".